### ISSUE TITLE
spyder crashes if ipython installed but not pygments

### ISSUE THREAD

**Issue by mikofski** [spyder-ide/spyder#2259]
Spyder 2.3.3
IPython 3

without pygments, spyder causes python to crash on windows,
hard for user to know what the issue is since there is no feedback,
and crash windows have strong negative association.
Spyder dialog says to use `spyder --reset`,
but that doesn't work since it's not the issue

please either
* make pygments required during installation
or
* handle ImportError gracefully
  - fall back to python interpreter console
  - issue message

Thanks!

**Comment by ccordoba12** [comment]
Please post the traceback you're seeing so we can understand how to fix it

**Comment by ccordoba12** [comment]
I'm almost sure this an IPython error rather than a Spyder one, but we can fix it in our own code.

**Comment by mikofski** [comment]
I'm sure it is IPython too, but it will turn away users from Spyder. The exception is `ImportError: No module named Pygments`

    Traceback (most recent call last):
      File "<xxx>", line X, in <module: xxx>
    ImportError: No module named Pygments

I was setting up python for a new engineer, and I forgot to install Pygments when setting up Python, and we started getting weird crashes everytime we started Spyder. it was frustrating because we had just run it before installing IPython, and I wanted to demo how spyder uses IPython console if installed. The error message from windows is not very useful. The impression some people get is that spyder is buggy unfortunately.
